Catappult/appcoins-wallet-android
# [APPC-3247] Notify users close to next level

**What does this PR do?**
Notifies users close to next level controlled by the BE. New field on user_stats: gamification_status.

**Database changed?**
No

**Where should the reviewer start?**
Start on PerkBonusAndGamificationService.kt

**How should this be manually tested?**
Topup until close to next level, make a purchase. The notification should show up.

**What are the relevant tickets?**
https://aptoide.atlassian.net/browse/APPC-3247

**Code Review Checklist**
